products/PurePhone/PurePhoneMain.cpp

```diff
@@ -192,7 +192,7 @@ int main()
                 platform->init();
             }
             catch (const std::runtime_error &e) {
-                LOG_FATAL("%s", e.what());
+                LOG_FATAL("Exception during platform init: %s", e.what());
```

Comment:
`LOG_ERROR` here

Reply:
I would consider keeping it as FATAL because after this log we have an abort() function

Reply:
Ok, sure